examples/classification/script.jl

```diff
@@ -0,0 +1,184 @@
+# # Stochastic Variational Classification
```

Comment:
This isn't actually _stochastic_ optimisation...:
```suggestion
# # 2. Classification: Sparse Variational Approximation for Non-Conjugate Likelihoods
```

Reply:
(NB- I'd prefer the more explicit headings even if you take out the numbering again)